Source/WebKit/UIProcess/API/Cocoa/NSAttributedString.mm

```diff
@@ -164,6 +184,45 @@ + (void)clearConfiguration
     globalConfiguration() = nil;
 }
 
++ (void)validateEntry:(id)maybeFileURL
+{
+    NSString *errorMessage = nil;
+
+    auto* url = dynamic_objc_cast<NSURL>(maybeFileURL);
+    if (!url)
+        errorMessage = @"The NSDictionary associated with _WKReadAccessFileURLsOption may only contain NSURL objects.";
+    else if (!url.fileURL)
```

Comment:
Isn’t this property named `isFileURL`? I’m surprised that `fileURL` exists.

Reply:
It's strange that it works -- I can see that the getter is declared "isFileURL", but this compiled. Strange -- I'll fix this. The test case properly threw with a non-FileURL, so I didn't notice.